level2/coco3/modules/boot_rom.asm

```diff
@@ -80,13 +89,17 @@ ExitOK              andcc     #^Carry             clear carry
 Uhoh                puls      u,y,x,a,b,cc,pc
 
 
-* this code executes at $4800
+* this code executes at RelAddy
 RelCode             equ       *
                     lda       #$4E                CC3 mode, MMU, 32K ROM
                     sta       $FF90
                     sta       $FFDE               ROM/RAM mode
 
 * Map ROM Blocks in
+                    ldd       $FFA6
```

Comment:
I thought that, in general, reading from $FFAx was not guaranteed to work
(and so the OS always keeps a shadow of these values in RAM somewhere). 

Not requiring this hardware ability (to read at least the low 6 bits from the MMU)
would allow simpler hardware designs for 6x09 SBCs to be coco-MMU compatible.

Reply:
I am almost certain that reads from $FFAx work fine. The OS doesn't keep shadows of $FFAx, although some registers in the $FF9x area are write only.